modules/ROOT/pages/download/nb26/index.adoc

```diff
@@ -0,0 +1,139 @@
+////
+     Licensed to the Apache Software Foundation (ASF) under one
+     or more contributor license agreements.  See the NOTICE file
+     distributed with this work for additional information
+     regarding copyright ownership.  The ASF licenses this file
+     to you under the Apache License, Version 2.0 (the
+     "License"); you may not use this file except in compliance
+     with the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing,
+     software distributed under the License is distributed on an
+     "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+     KIND, either express or implied.  See the License for the
+     specific language governing permissions and limitations
+     under the License.
+////
+////
+
+NOTE: 
+See https://www.apache.org/dev/release-download-pages.html 
+for important requirements for download pages for Apache projects.
+
+////
+= Downloading Apache NetBeans 26
+:page-layout: page_noaside
+:page-tags: download
+:jbake-status: published
+:keywords: Apache NetBeans 25 Download
+:description: Apache NetBeans 25 Download
+:toc: left
+:toc-title:
+:icons: font
+
+// check version above and follow the TODOs
+:netbeans-version: 26
+// TODO            ^^
+
+// https://infra.apache.org/release-distribution.html#download-links
+// TODO base URLs - when archiving, delete the next 2 lines and uncomment the following 2
+:url-download: https://www.apache.org/dyn/closer.lua/netbeans/
+:url-download-keychecksum: https://downloads.apache.org/netbeans/
+// :url-download: https://archive.apache.org/dist/netbeans/
+// :url-download-keychecksum: https://archive.apache.org/dist/netbeans/
+
+//// 
+url-download depends of release status archived or not
+https://archive.apache.org/dist/netbeans/  (//archived)
+https://www.apache.org/dyn/closer.lua/netbeans/ (//current)
+
+url-download-keychecksum depends of release status archived or not
+https://archive.apache.org/dist/netbeans/  (//archived)
+https://downloads.apache.org/netbeans/ (//current)
+////
+
+Apache NetBeans {netbeans-version} was released on May xxx, 2025.
+// TODO update date                                ^^^^
+
+////
+NOTE: It's mandatory to link to the source. It's optional to link to the binaries.
+NOTE: It's mandatory to link against https://www.apache.org for the sums & keys. https is recommended.
+NOTE: It's NOT recommended to link to github.
+////
+
+*Binaries (Platform Independent):*
+
+* link:{url-download}netbeans/{netbeans-version}/netbeans-{netbeans-version}-bin.zip?action=download[netbeans-{netbeans-version}-bin.zip]
```

Comment:
Could be nice to have the links for zips and to packages as buttons?  Can we add classes (`.button`, `.success`, etc.) to links using this syntax?

Reply:
I don't know but probably. I am not sure if this would look good if the file name text is in the button. They typically look good if they have all the same size like on https://netbeans.apache.org/download/archive/